streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -374,7 +374,17 @@ abstract class DStream[T: ClassTag] (
    * Wrap a body of code such that the call site and operation scope
    * information are passed to the RDDs created in this body properly.
    */
-  protected def createRDDWithLocalProperties[U](time: Time)(body: => U): U = {
+  protected[streaming] def createRDDWithLocalProperties[U](time: Time)(body: => U): U = {
```

Comment:
mima only checks the binary signature. `protected[streaming]` doesn't change that. Both of them are just `public` in the jvm method signature.